dangerfile.js

```diff
@@ -0,0 +1,25 @@
+import { message, danger } from 'danger';
```

Comment:
<!--
  0 failure: 
  0 warning: 
  1 messages
  
  DangerID: danger-id-default;
  File: dangerfile.js;
  Line: 1;
-->  


- :book: 'danger' should be listed in the project's dependencies, not devDependencies. (import/no-extraneous-dependencies)